magento/pwa-studio
# Fixed #2426 Error if product is out of stock

## Description

Fixed #2426 Error if product is out of stock

## Related Issue





Closes #2426

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Go to Product Page of any out of stock product
2. Make sure in magento backend the below setting is set to `NO` 
    Store->Configuration->Catalog->Inventory->Display Out of Stock Products
3. Error on Product Page 

## Screenshots / Screen Captures (if appropriate)
![image](https://user-images.githubusercontent.com/25526037/82727804-5c2e6680-9d0a-11ea-8c8a-3bb1cdb539dd.png)


## Checklist

* I have updated the documentation accordingly, if necessary.
[ ] I have added tests to cover my changes, if necessary.
